rrmngmnt/service.py

```diff
@@ -211,7 +211,8 @@ def disable(self):
         return self._execute('disable')
 
     def status(self):
-        return self._execute('status')
+        out = self._execute('status')
```

Comment:
such change changes interface, I can not really include such change in repository because it will break all code which uses this module ...

what you can do is create new method for you

Reply:
Fixed